docs/setup.rst

```diff
@@ -11,29 +11,34 @@ To setup the simulator, which is called SMARTS, run the following commands,
 
 .. code-block:: bash
 
-    # unzip the starter_kit and place somewhere convenient on your machine. (e.x. ~/src/starter_kit)
+    # For Mac OS X users, make sure XQuartz is pre-installed as SUMO's dependency
```

Comment:
Though this is true, sumo will prompt them about this. Adding it here makes install look more complicated :)